public/css/app.css

```diff
@@ -11061,6 +11061,66 @@ a.text-dark:focus {
   border-radius: 3px;
 }
 
+.job-lists {
+  padding: 16px 25px;
+  position: relative;
+  display: flex;
+  border: 1px solid #e5e5e5;
+  background: #fff;
+  overflow: hidden;
+  transition: all 0.4s ease-in-out;
+  border-radius: 3px;
+}
+
+.logo-section {
+  width: 20%;
+}
+
+.job-list-section {
+  width: 100%;
+  padding: 0px 0px 0px 30px;
+}
+
+.job-title {
+  color: #fa590f;
+  font-family: "Work Sans", sans-serif;
+  font-size: 20px;
+}
+
+.company-info {
+  color: #9f9f9f;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  padding-bottom: 3px;
+}
+
+.company-name {
+  margin: 0px 9px 0px 0px;
+}
+
+.company-info-details {
+  color: #9f9f9f;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  padding-bottom: 3px;
+}
+
+.location {
+  margin: 3px 24px 0px 0px;
+}
+
+.salary {
+  margin: 3px 24px 0px 0px;
```

Comment:
Can you create one generic class for both location and salary as both are having the same style

Reply:
Done